action/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "DDG Sync GitHub Action",
   "main": "index.js",
   "engines": {
-    "node": "^18"
+    "node": "^20"
```

Comment:
we're already running `node20` on https://github.com/duckduckgo/sync_crypto/blob/4e383fa27d3386a48e01a43fd890540a7d8656aa/action/action.yml#L21